pages/content/amp-dev/events/amp-conf-2019.html

```diff
@@ -53,15 +53,22 @@ <h2 class="ap-o-stage-content-subline">ようこそ from Tokyo, Japan</h2>
   <div class="ap--content-x-wide">
     <h2 class="ap-o-video-slider-text">Discover all exciting news from Day 1</h2>
     <p class="ap-o-video-slider-text">You couldn't make it to Tokyo but still want to know about all the exciting things happening around AMP? Checkout the highlights of Day 1 and even witness the AMP team unleashing the kraken!</p>
-    {% with %}
-    {% set youtube_video_ids = [
-      '1vwOFt9FBm4',
-      'sYXkVOiz77I',
-      'BrDBsS1Sko0',
-      ]
-    %}
-    {% include '/views/partials/video-carousel.j2' %}
-    {% endwith %}
+    [% set youTubeChannelPosts = []|importYouTubeChannel %]
+    [% if youTubeChannelPosts|length %]
+    <amp-carousel class="ap-o-video-slider-carousel"
+        type="carousel"
+        height="300"
+        layout="fixed-height"
+        controls=0>
+      [% for post in youTubeChannelPosts %]
+      <amp-youtube data-videoid="[=post.id=]"
+          width="391"
+          height="220"
+          layout="fixed">
+      </amp-youtube>
+      [% endfor %]
+    </amp-carousel>
+    [% endif %]
```

Comment:
Similar to the courses: this is a specific set of videos. Either check if there's an RSS feed for a specific playlist (in this case this one: https://www.youtube.com/watch?v=1vwOFt9FBm4&list=PLXTOW_XMsIDSY0USlzgoaIkRyPcHklrEl) or revert to the previous solution.